src/extensions/scratch3_vizblocks/index.js

```diff
@@ -544,8 +550,8 @@ class Scratch3VizBlocks {
                     arguments: {
                         CHART: {
                             type: ArgumentType.STRING,
-                            menu: 'LINE_SCATTER_CHART',
-                            defaultValue: 'line chart'
+                            menu: 'Y_CHART_TYPE',
+                            defaultValue: 'dot plot'
                         },
```

Comment:
The y-axis option doesn't have dot plot, in this case we can default to line chart